scss/mixins.scss

```diff
@@ -20,262 +17,106 @@ $_o-brand-default-not-set-warning: false !default; // Has the default not set wa
     @if type-of($brand) != 'string' or $brand == '' {
         @error 'Could not set brand "#{$brand}" for component "#{$component}". A brand of type string must be given.';
     }
-    // Define brand configuration.
-    @if not _oBrandIsDefined($component, $brand) {
-        $result: _oBrandUpdateConfig($component, $brand, $config);
+    // Do not allow a component brand to be overridden.
+    @if _oBrandIsDefined($component, $brand) {
+        @error 'Brand #{$brand} is already defined for #{$component}.';
```

Comment:
👍

Reply:
I'm not sure when reconfiguring a brand for a component would happen, brand configuration for a component has always been done within the component's sass itself.